custom-recipes/nlp-analysis-ontology-tagging/recipe.json

```diff
@@ -347,8 +347,8 @@
             "type": "SEPARATOR"
         },
         {
-            "name": "case_insensitive",
-            "label": "Case-insensitive",
+            "name": "case_insensitivity",
+            "label": "Case-insensitivity",
```

Comment:
Nitpick: perhaps more natural to order matching parameters from simple to advanced? 1. Case-insensitivity 2. Normalization 3. Lemmatization?

Reply:
Good idea, done in 961a8cf